lib/wsdl.js

```diff
@@ -1403,7 +1403,9 @@ WSDL.prototype.xmlToObject = function(xml, callback) {
   };
 
   p.oncdata = function (text) {
-    text = trim(text);
+    if (!self.options || !self.options.preserveWhitespace) {
```

Comment:
I guess it's save to check for `self.options.preserveWhitespace` only as the options should be set up anyways(?)